src/Service/AnrInstanceService.php

```diff
@@ -892,6 +892,9 @@ public function importFromArray($data, $anr, $idParent = null, $modeImport = 'me
             }
 
             if (!empty($data['risksop'])) {
+              usort($data['risksop'], function($a,$b){
+                return $a['rolfRisk'] <=> $b['rolfRisk'];
+              });
```

Comment:
We need to follow, at least for new code the PSR-2 standards, https://www.php-fig.org/psr/psr-2/, otherwise the code is hard to maintain and read.
- "Code MUST use 4 spaces for indenting, not tabs."
- space before opening curve brace `{`
- space after comma, to separate variables.